guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -72,7 +73,8 @@ For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote
 # subscription-manager refresh
 ----
 +
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
-. Locate {SmartProxyServer} in the list, and click *Edit* to the right of it.
-. Edit the *Name* and *URL* fields to match {SmartProxyServer}'s new host name, then click *Submit*.
-. On your DNS server, add a record for {SmartProxyServer}'s new host name, and delete the record for the previous host name.
+. Update the {SmartProxy} host name in the {ProjectWebUI}.
+.. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+.. Locate {SmartProxyServer} in the list, and click *Edit*.
+.. Edit the *Name* and *URL* fields to match {SmartProxyServer}'s new host name, then click *Submit*.
+.. On your DNS server, add a record for {SmartProxyServer}'s new host name, and delete the record of the previous host name.
```

Comment:
Unsure, but I lean towards "... the new host name of your {SmartProxyServer} ..." because it's a thing, not a person.